plantdescriptionengine/pom.xml

```diff
@@ -14,7 +14,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.release>11</maven.compiler.release>
-        <kalix.version>0.6.0</kalix.version>
+        <kalix.version>0.6.1</kalix.version>
         <mysql.jdbc.driver>8.0.16</mysql.jdbc.driver>
     </properties>
 
```

Comment:
Tests failed in maven because of referencing a junit jupiter class in CodecExceptionCatcherTest. I think the classpath does not contain the junit5, only junit4. I fix it in my environment by replacing the jupiter's assertEquals with the junit4's equivalent. 

You can do the same or just add the necessary dependency.